backend/vaa-strapi/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
   "extends": "@strapi/typescript-utils/tsconfigs/server",
   "compilerOptions": {
-    "moduleResolution": "Node10",
```

Comment:
Removing this bc the deprecated modRes is defined in @strapi/typescript-utils/tsconfigs/server. When we update Strapi, we don't want this override to remain.